### ISSUE TITLE
Navigational direction setting inaccessible via tabbing

### ISSUE THREAD

**Issue by GreenReaper** [mucklet/mucklet-client#324]
In the exit edit UI, when tabbing from Keywords, I expected to get to Navigation. However, I was brought directly to Icon, and Navigation seems not to be in the tab order at all, hence inaccessible to those relying on keyboard input.

I notice that it uses 'use' and 'g' SVG elements rather than buttons like the icon selector. [As suggested here](https://css-tricks.com/accessible-svgs/#aa-2-making-it-accessible) I added `<a xlink:href="#btn" role="img" class="navbuttons--btn dir-n" tabindex="0" role="listitem"></a>` after the `<g>` and this became selectable in the tab order but it did not provide a highlight effect and activating the anchor had no effect. Presumably it might work better if hooked up appropriately.

I'm not sure if `listitem` is the correct role, or if the SVG parent requires `list` or `group`.

**Comment by anisus** [comment]
I just tried out your suggestion, and also tried to use `:focus` styling for the `navbuttons--btn` classes.
After a bit of trial and error, I got it to work in an  manner. At least on Chrome.

My solution simply was not to add any anchor tag, but instead I just added `tabindex="0"` to the `<g>` tags. I also added a `keydown` handler to handle _space_ and _enter_ events.
Let's try it out.

Thanks for the effort, @GreenReaper!
